### PR TITLE
Add clarifications to board and serial number descriptions

### PR DESCRIPTION
**Pull request by aravindksg**
Addresses #179

### CODE REVIEW

**Comment by aravindksg** [comment]
@jandres742 @saik-intel  @vaidyan1-int @wdamon-intel for review

**Review by jandres742 — APPROVED** [review]

File: scripts/sysman/device.yml [hunk 1]
```diff
@@ -192,10 +192,10 @@ members:
       desc: "[out] Number of sub-devices. A value of 0 indicates that this device doesn't have sub-devices."
     - type: char
       name: "serialNumber[$S_STRING_PROPERTY_SIZE]"
-      desc: "[out] Manufacturing serial number (NULL terminated string value). Will be set to the string \"unkown\" if this cannot be determined for the device."
+      desc: "[out] Manufacturing serial number (NULL terminated string value). This value is intended to reflect the Part ID/SoC ID assigned by manufacturer that is unique for a SoC. Will be set to the string \"unkown\" if this cannot be determined for the device."
```
Comment (wdamon-intel) [inline scripts/sysman/device.yml:195]: Might as well fixup the spelling for "unkown" -> "unknown", too.

File: scripts/sysman/device.yml [hunk 2]
```diff
@@ -192,10 +192,10 @@ members:
...
     - type: char
       name: "boardNumber[$S_STRING_PROPERTY_SIZE]"
-      desc: "[out] Manufacturing board number (NULL terminated string value). Will be set to the string \"unkown\" if this cannot be determined for the device."
+      desc: "[out] Manufacturing board number (NULL terminated string value). Alternatively \"boardSerialNumber\", this value is intended to reflect the string printed on board label by manufacturer. Will be set to the string \"unkown\" if this cannot be determined for the device."
```
Comment (wdamon-intel) [inline scripts/sysman/device.yml:198]: same

**Review by wdamon-intel — CHANGES_REQUESTED** [review]
There's an opportunity here to fix-up a couple of spelling errors.

File: scripts/sysman/device.yml (hunk 2 above)
Comment (aravindksg) [reply scripts/sysman/device.yml]: done

**Review by wdamon-intel — APPROVED** [review]
LGTM

**Pull request merged**